<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kutemail</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnCompose">
        <property name="text">
         <string>Compose</string>
        </property>
        <property name="icon">
         <iconset theme="document-new">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeMailWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listEmails"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="emailPreview"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCompose"/>
   <addaction name="actionRefresh"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionCompose">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Compose</string>
   </property>
   <property name="toolTip">
    <string>Write new eMail</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Check for new eMails</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCompose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onComposeMail()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRefresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>682</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCompose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onComposeMail()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeMailWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onFolderSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listEmails</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onMailSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onComposeMail()</slot>
  <slot>onRefresh()</slot>
  <slot>onFolderSelected()</slot>
  <slot>onMailSelected()</slot>
 </slots>
</ui>
